docs/subject/subject-schema.yaml

```
@@ -41,7 +43,6 @@ definitions:
       name:
         type: string
       category:
-        type: string
         $ref: '#/components/category'
```

```suggestion
        $ref: '#/definitions/resourcesCategory'
```
but about resourcesCategory instead of category i am not sure